docs/README.mdx

```diff
@@ -1,18 +1,46 @@
+import DocCard from '@theme/DocCard';
 import DocCardList from '@theme/DocCardList';
 import {findSidebarItem} from '@site/src/sidebarUtils';
 
-# Getting Started
+# Oasis Platform Documentation
 
 ## Use Oasis
 
-These guides provide general overview of the Oasis Network and introduce basic
-tools for you to get started.
+The first section contains general overview of the Oasis Network such as the
+distinction between the consensus layer and different ParaTimes. We will also
+cover wallets and other tools for managing your assets across the Oasis chains
+and use unique Oasis features.
 
 <DocCardList items={[
     findSidebarItem('/general/oasis-network/'),
     findSidebarItem('/general/manage-tokens/'),
 ]} />
 
+## DApp Developers
```

Comment:
I would rename this to "Create dApp" to be consistent with the above header.
And we also avoid the ambiguous "DApp" vs. "dApp" capitalization.